R/adducts.R

```diff
@@ -1,18 +1,47 @@
-#' @title Create list with all adduct calculation rules (all rules)
-#' 
-#' This function returns a list with rules for the calculation of adducts. 
-#' It is required for the calculation of adduct m/z values from exact masses 
-#' and the other way round. This list contains all rules for both ionization 
-#' modes.
-#' 
-#' 
-#' @author Michael Witting, \email{michael.witting@@helmholtz-muenchen.de}
-#'
-#' @seealso \code{\link{adduct_rules_pos}}
-#' @seealso \code{\link{adduct_rules_neg}}
-#' @seealso \code{\link{get_adduct_names}}
-#'
-#' @export
+mass2mz <- function(exact_mass, adduct) {
```

Comment:
please change parameter `exact_mass` to `x`. Also, it is usually better to provide a defaults for parameters if possible. For `adduct` you could use `adduct = names(adductRules())`.